build.gradle

```diff
@@ -44,7 +44,7 @@ subprojects {
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '2.9'
+    gradleVersion = '2.13'
```

Comment:
That will update the URL to 'bin' instead of 'all'.

On Fri, Jun 17, 2016 at 10:22 AM Roman Mazur <notifications@github.com>
wrote:

> In build.gradle
> <https://github.com/JakeWharton/RxBinding/pull/256#discussion_r67517759>:
>
> > @@ -44,7 +44,7 @@ subprojects {
> >  }
> >
> >  task wrapper(type: Wrapper) {
> > -    gradleVersion = '2.9'
> > +    gradleVersion = '2.13'
>
> btw, this task configuration is not required any more
> Since some 2.x Gradle version, wrapper is a task added to every project
> and you can update the wrapper with
>
> ./gradlew wrapper --gradle-version 2.0
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/JakeWharton/RxBinding/pull/256/files/8276c1d29316beab5d64ac95593092593559da92#r67517759>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAEEEeB_S-t_nDbOwL8MDr0wozqiinukks5qMq2ogaJpZM4IvL3D>
> .
>